.eslintrc.cjs

```diff
@@ -108,6 +108,8 @@ module.exports = defineConfig({
     ],
 
     'regexp/no-contradiction-with-assertion': 'error',
+    // in some cases using explicit letter-casing is more performant than the `i` flag
+    'regexp/use-ignore-case': 'off',
```

Comment:
Checked https://stackoverflow.com/questions/32010/is-regex-case-insensitivity-slower and I think we should ignore this rule for now. It's suggesting our `[a-zA-Z]` regexes to use the `i` flag.